### PR TITLE
Python 3: Add Travis-CI checks (allowing failure)

### PR DESCRIPTION
**Pull request by clebergnu**
And also drop the 2.6 checks.

### CODE REVIEW

**Comment by clebergnu** [comment]
[Broken python2.6 check on avocado-vt?](https://trello.com/c/vLpwbwH5/1125-broken-python26-check-on-avocado-vt)

File: .travis.yml [hunk 1]
```diff
@@ -1,14 +1,18 @@
 language: python
 python:
     - "2.7"
-    - "2.6"
+    - "3.4"
+
+matrix:
+  allow_failures:
+    - python: "3.4"
```
Comment (chunfuwen) [inline .travis.yml]: I assume this is temporary config for adapting python3.4 porting

Comment (clebergnu) [reply .travis.yml]: @chunfuwen exactly!

**Comment by balamuruhans** [comment]
Thanks all, Let us merge this.

**Pull request merged**